.gitignore

```diff
@@ -21,6 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```

Comment:
I see that you've deleted `package-lock.json` - you should keep it around! Here's a short summary: https://stackoverflow.com/questions/44297803/what-is-the-role-of-the-package-lock-json

Reply:
Gonna use yarn instead, so I'll commit that one